src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -2913,12 +2913,16 @@ class run::command_error_impl
 // class runlist_impl - The internals of a runlist
 class runlist_impl
 {
+  static constexpr size_t submit_size = 25;
```

Comment:
Let's use 24. We mis-calculated it. @mamin506 , please confirm.

Reply:
Sure, done.